README.md

```diff
@@ -222,6 +222,7 @@ Additional to the Filecoin FFI Git submodules, we are also bundling `proof param
 * `v4` => locking in Filecoin FFI commit 6a143e06f923f3a4f544c7a652e8b4df420a3d28.
 * `v5` => locking in Filecoin FFI commit cddc56607e1d851ea6d09d49404bd7db70cb3c2e.
 * `v6` => locking in Filecoin FFI commit 40569104603407c999d6c9e4c3f1228cbd4d0e5c.
+* `v7` => add Filecoin-BLST repo to buildbase.
```

Comment:
* should we track the BLST commit hash here in the same way as the FFI one?
* is this gonna imply that lotus in submoduling oni now needs to keep versions for both of these in sync?

Reply:
> should we track the BLST commit hash here in the same way as the FFI one?

I think so. I don't see a way around it.

> is this gonna imply that lotus in submoduling oni now needs to keep versions for both of these in sync?

Yes. However since Magik also requested that we use Lotus' version of FFI, I added replaces for all the dependencies. This means that when compiling `lotus-soup` in the `lotus` repo, we are always using the `lotus` dependencies and make sure it compiles (to me this sounds like less overhead to the Lotus team).

Obviously we would need to on-demand update versions in `oni/lotus-soup` itself (for example here)...

---

I don't like this, although I don't have any better ideas on how to improve right now.